common/tests/fixtures/ethereum/signmessage.json

```diff
@@ -5,72 +5,80 @@
+                "address": "0x73d0385F4d8E00C5e6504C6030F47BF6212736A8",
+                "path": "44'/60'/0'/0/0"
             },
             "result": {
                 "sig": "e9b3648fe10146a7c5c73ae1e42468273b945554b13769be19c3658e5b55c62d7036b56c1e094dc364060b798b120a528f31d91e815d0612c367bf7471519c761c"
             }
         },
         {
-            "name": "3",
             "parameters": {
-                "msg": "MsgLenIs9"
+                "msg": "MsgLenIs9",
+                "address": "0x73d0385F4d8E00C5e6504C6030F47BF6212736A8",
+                "path": "44'/60'/0'/0/0"
             },
             "result": {
                 "sig": "495b5b021517cd01cb05f02dfb6a1f79a8d9e4f82e8fb01d44ebf7d1218333f1528b67043ffc25c5b64b1e5a182f5d7d5707fcb639092af708969f90038155a91c"
             }
         },
         {
-            "name": "4",
             "parameters": {
-                "msg": "MsgLenIs10"
+                "msg": "MsgLenIs10",
+                "address": "0x73d0385F4d8E00C5e6504C6030F47BF6212736A8",
+                "path": "44'/60'/0'/0/0"
             },
             "result": {
                 "sig": "0c8eb57decbb95d2a783e69d444b3747c49ebcceb14f24287732ba77c39aa5cd2c66f7ec545b8108ef7ce078edac4588c70ab6249fc626114a95606783828f041b"
             }
         },
         {
-            "name": "5",
             "parameters": {
-                "msg": "MsgLenIs11!"
+                "msg": "MsgLenIs11!",
+                "address": "0x73d0385F4d8E00C5e6504C6030F47BF6212736A8",
+                "path": "44'/60'/0'/0/0"
             },
             "result": {
-                "sig": "f2c3d9c81fe35a6bc68f0ace01bd7209baf11e455e7335903e7ad2879ce5d7bc2199bf8bafc1efe1ff2cfdf2963cd5767911a641c9445cca6f37dc6da72fd1611b"
+                "sig": "5f1ab8940c2137baf69a2b94f55f0f7e53a022b9e363b564744616271eb89b66145055fc9923c6ead63dd784854c742f4f5651c8257d70cbce5a96ce2986dd311c"
             }
         },
         {
-            "name": "6",
             "parameters": {
-                "msg": "This message has length 99!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+                "msg": "This message has length 99!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!",
+                "address": "0x73d0385F4d8E00C5e6504C6030F47BF6212736A8",
+                "path": "44'/60'/0'/0/0"
             },
             "result": {
                 "sig": "f2c3d9c81fe35a6bc68f0ace01bd7209baf11e455e7335903e7ad2879ce5d7bc2199bf8bafc1efe1ff2cfdf2963cd5767911a641c9445cca6f37dc6da72fd1611b"
             }
         },
         {
-            "name": "7",
             "parameters": {
-                "msg": "This message has length 100!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+                "msg": "This message has length 100!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!",
+                "address": "0x73d0385F4d8E00C5e6504C6030F47BF6212736A8",
```

Comment:
the `address` is not an input, it's a result

Reply:
Fixed in 59c7eb7